### DISCUSSION TITLE
Writing 40 track DD disks on a 80 track drive (for the PC-8801 and likely more)

### DISCUSSION THREAD

**Discussion by Pawlicker** [keirf/greaseweazle#419] [Show and tell]
This should be on the Wiki honestly, because I struggled to find this information with googling.

If you want to read and write a 5.25 disk that's double density, you’ll need to use a special command. But first, you’ll want to google or look on bitsavers for your floppy drive’s datasheet. Make sure you have the proper jumper set for DD operation, for example on a Mitsubishi drive I had to set my drive to function in 300rpm and 360rpm mode depending on if DD mode was selected. Then; you will want to issue this command:

`gw read –tracks c=0-39:h=0,1:step=2 –dd=L yourimage.scp`

The most important part is the step command, along with the sectors you’re reading. The other important command is –dd=L, this sets pin 2 as low to enable double density reading and let the drive know “hey, this is a DD disk”. This is because 5.25 stupidly had no sense hole like 3.5 did later on. The step command tells the drive to “double step”, and read out a 40 track disk on an 80 track drive. When writing these disks, you will want to do the same:

`gw write –tracks c=0-39:h=0,1:step=2 –dd=L –pre-erase`

This does the same, along with erasing any failed attempts at writing the disk. You’ll probably want to use “gw erase” to wipe the disk as well in this case. 

This is what I had to do to make PC-8801 40 track DD disks on my 80 track drive.

**Comment by keirf** [comment]
I'll have a think how to merge this in. The problem is making this info easily findable. A lot of it is in there *somewhere* in the wiki, for example in Yan Serra's tutorial https://github.com/keirf/greaseweazle/wiki/Yann-Serra-Tutorial#level-4-manage-525-inch-48-tpi-disks-with-your-525-inch-96-tpi-drive

**Reply by wohali** [reply]
While credit is certainly due to Yann Serra, I suggest renaming the title of that page - it's an obscure heading in the listing.
